### PR TITLE
remove-obsolete-brms-refs 

### PR DESCRIPTION
**Pull request by emmurphy1**
Because of the number of merge conflicts in https://github.com/redhat-documentation/supplementary-style-guide/pull/348, I created a new branch and addressed all of the comments in the previous PR.

### CODE REVIEW

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc [hunk 1]
```diff
@@ -578,13 +578,21 @@ Red Hat Trusted Application Pipeline users can configure secrets to inject sensi
 
 [[intel-64]]
 ==== image:images/yes.png[yes] Intel 64 (noun)
+<<<<<<< HEAD
```
Comment (emmurphy1) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:581]:
```suggestion
```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc [hunk 2]
```diff
@@ -579,12 +579,19 @@ Red Hat Trusted Application Pipeline users can configure secrets to inject sensi
 [[intel-64]]
 ==== image:images/yes.png[yes] Intel 64 (noun)
 *Description*: The Intel 64-bit version of the x86 architecture. Use this format when referring to information that is exclusive to Intel processors. Use only in RHEL documentation.
+=======
+*Description*: The Intel 64-bit version of the x86 architecture. Use this format when referring to information that is exclusive to Intel processors. Only use in RHEL documentation.
```
Comment (emmurphy1) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:583]:
```suggestion
```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc [hunk 3]
```diff
@@ -579,12 +579,17 @@ Red Hat Trusted Application Pipeline users can configure secrets to inject sensi
 [[intel-64]]
 ==== image:images/yes.png[yes] Intel 64 (noun)
 *Description*: The Intel 64-bit version of the x86 architecture. Use this format when referring to information that is exclusive to Intel processors. Use only in RHEL documentation.
-
+=======
```
Comment (emmurphy1) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:582]:
```suggestion
```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc [hunk 4]
```diff
@@ -209,7 +209,10 @@ _This Operator is supported on AMD64 and ARM64 platforms._
 
 *Use it*: yes
 
+<<<<<<< HEAD
```
Comment (emmurphy1) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc:212]:
```suggestion
<<<<<<< HEAD
```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc [hunk 5]
```diff
@@ -209,7 +209,10 @@ _This Operator is supported on AMD64 and ARM64 platforms._
...
 [.vale-ignore]
+=======
```
Comment (emmurphy1) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc]:
```suggestion
```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc [hunk 6]
```diff
@@ -209,7 +209,9 @@ _This Operator is supported on AMD64 and ARM64 platforms._
 
 *Use it*: yes
 
+<<<<<<< HEAD
```
Comment (emmurphy1) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc:212]:
```suggestion
```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc [hunk 7]
```diff
@@ -210,6 +210,7 @@ _This Operator is supported on AMD64 and ARM64 platforms._
 *Use it*: yes
 
 [.vale-ignore]
+>>>>>>> 99dc3c6 (remove-obsolete-brms-refs updated doc)
```
Comment (emmurphy1) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc:213]:
```suggestion
>>>>>>> 99dc3c6 (remove-obsolete-brms-refs updated doc)
```

Comment (emmurphy1) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc:213]:
```suggestion
>>>>>>> 99dc3c6 (remove-obsolete-brms-refs updated doc)
```

Comment (emmurphy1) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc:213]:
```suggestion
>>>>>>> 99dc3c6 (remove-obsolete-brms-refs updated doc)
```
```suggestion
>>>>>>> 99dc3c6 (remove-obsolete-brms-refs updated doc)
```

Comment (emmurphy1) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc:213]:
```suggestion
```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc [hunk 8]
```diff
@@ -360,9 +361,16 @@ Always use the approved product name _Red Hat build of OpenJDK_ instead.
 *See also*:
 
 // BxMS: General; kept as is
+<<<<<<< HEAD
```
Comment (emmurphy1) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:364]:
```suggestion
```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc [hunk 9]
```diff
@@ -579,12 +579,14 @@ Red Hat Trusted Application Pipeline users can configure secrets to inject sensi
...
 *Description*: The Intel 64-bit version of the x86 architecture. Use this format when referring to information that is exclusive to Intel processors. Use only in RHEL documentation.
-
 Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
 
 Example:
 
 _This feature can run only on Intel 64 processors_
+=======
+_This feature can run on only Intel 64 processors_
+>>>>>>> 99dc3c6 (remove-obsolete-brms-refs updated doc)
```
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:589]: >>>>>>> 99dc3c6 (remove-obsolete-brms-refs updated doc) needs to be removed.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc [hunk 10]
```diff
@@ -363,6 +364,12 @@ Always use the approved product name _Red Hat build of OpenJDK_ instead.
 [[brms]]
 ==== image:images/yes.png[yes] Red Hat JBoss BRMS (noun)
 *Description*: _Red Hat JBoss BRMS_ is a comprehensive platform for business rules management, business resource optimization, and complex event processing (CEP). BRMS stands for Business Rules Management System. Organizations can use Red Hat JBoss BRMS to incorporate sophisticated decision logic into line-of-business applications and quickly update underlying business rules as market conditions change.
+=======
```
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:367]: @emmurphy1 Seeing some conflict markup showing here that needs to be removed

```suggestion

```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc [hunk 11]
```diff
@@ -363,6 +364,12 @@ Always use the approved product name _Red Hat build of OpenJDK_ instead.
...
+[discrete]
+[[rhdm]]
+==== image:images/yes.png[yes] Red Hat Decision Manager (noun)
+*Description*: _Red Hat Decision Manager_ is a comprehensive platform for business rules management, business resource optimization, and complex event processing (CEP). Red Hat Decision Manager is a subset of Red Hat Process Automation Manager. Organizations can use Red Hat Decision Manager to incorporate sophisticated decision logic into line-of-business applications and quickly update underlying business rules as market conditions change.
+>>>>>>> 99dc3c6 (remove-obsolete-brms-refs updated doc)
```
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:372]:
```suggestion

```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc [hunk 12]
```diff
@@ -579,12 +579,14 @@ Red Hat Trusted Application Pipeline users can configure secrets to inject sensi
 [[intel-64]]
 ==== image:images/yes.png[yes] Intel 64 (noun)
 *Description*: The Intel 64-bit version of the x86 architecture. Use this format when referring to information that is exclusive to Intel processors. Use only in RHEL documentation.
-
 Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
 
 Example:
 
 _This feature can run only on Intel 64 processors_
+=======
```
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:587]:
```suggestion

```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc (hunk 9 above)
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:589]:
```suggestion

```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc [hunk 13]
```diff
@@ -363,6 +364,12 @@ Always use the approved product name _Red Hat build of OpenJDK_ instead.
 [[brms]]
 ==== image:images/yes.png[yes] Red Hat JBoss BRMS (noun)
```
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:365]: Just want to check that the "Red Hat JBoss BRMS" is still required? Just nopticed that Git has added the full entry in the comparison tool.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc [hunk 14]
```diff
@@ -349,8 +349,9 @@ Always use the approved product name _Red Hat build of OpenJDK_ instead.
 
 // BxMS: General; kept as is
 [[bpms]]
-==== image:images/yes.png[yes] Red Hat JBoss BPM Suite (noun)
-*Description*: _Red Hat JBoss BPM Suite_ is the JBoss platform for Business Process Management (BPM). The Red Hat JBoss BPM Suite enables enterprise business and IT users to document, simulate, manage, automate, and monitor business processes and policies. It is designed to empower business and IT users to collaborate more effectively, so business applications can be changed more easily and quickly.
+==== image:images/yes.png[yes] Red Hat Process Automation Manager (noun)
```
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:352]: This needs to be moved to be in alphabetical order

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc [hunk 15]
```diff
@@ -363,6 +364,12 @@ Always use the approved product name _Red Hat build of OpenJDK_ instead.
 [[brms]]
 ==== image:images/yes.png[yes] Red Hat JBoss BRMS (noun)
 *Description*: _Red Hat JBoss BRMS_ is a comprehensive platform for business rules management, business resource optimization, and complex event processing (CEP). BRMS stands for Business Rules Management System. Organizations can use Red Hat JBoss BRMS to incorporate sophisticated decision logic into line-of-business applications and quickly update underlying business rules as market conditions change.
+=======
+[discrete]
+[[rhdm]]
+==== image:images/yes.png[yes] Red Hat Decision Manager (noun)
```
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:370]: This isn't in alphabetical order now

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc [hunk 16]
```diff
@@ -363,6 +364,12 @@ Always use the approved product name _Red Hat build of OpenJDK_ instead.
 [[brms]]
 ==== image:images/yes.png[yes] Red Hat JBoss BRMS (noun)
 *Description*: _Red Hat JBoss BRMS_ is a comprehensive platform for business rules management, business resource optimization, and complex event processing (CEP). BRMS stands for Business Rules Management System. Organizations can use Red Hat JBoss BRMS to incorporate sophisticated decision logic into line-of-business applications and quickly update underlying business rules as market conditions change.
```
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:366]: Missing ?

*Use it*: yes

[.vale-ignore]
*Incorrect forms*: BRMS, BRM, JBoss BRMS

*See also*:

**Review by bergerhoffer — COMMENTED** [review]
Some conflict markup showing in the source, and a few items need to be alphabetized properly. Getting close!

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc (hunk 11 above)
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:372]: Marker need to be deleted.

Comment (dfitzmau) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:372]: I do not see this entry in the [upstream version](https://github.com/redhat-documentation/supplementary-style-guide/blob/main/supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc) of the file. Just want to cehck that it's a new entry?

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc (hunk 13 above)
Comment (emmurphy1) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:365]: Not required. I'll check the source.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc (hunk 16 above)
Comment (emmurphy1) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:366]: Not missing.

**Comment by emmurphy1** [comment]
Closed due to merge conflicts. Please review https://github.com/redhat-documentation/supplementary-style-guide/pull/462.

**Pull request closed without merging**